x-pack/plugins/observability_solution/slo/docs/openapi/slo/bundled.yaml

```diff
@@ -713,17 +745,16 @@ components:
               description: The index or index pattern to use
               type: string
               example: my-service-*
+            dataViewId:
+              description: The kibana data view id to use, primarily used to include data view runtime mappings. Make sure to save SLO again if you add/update run time fields to the data view and if those fields are being used in slo queries.
+              type: string
+              example: 03b80ab3-003d-498b-881c-3beedbaf1162
```

Comment:
Out of curiosity, since this is of type string, shouldn't it be in quotes `'03b80ab3-003d-498b-881c-3beedbaf1162'`

Reply:
That seems to be the way strings are handled in these yaml files. I added single quotes in the source files and they still get output like this.